src/ra_log_snapshot.erl

```diff
@@ -1,9 +1,15 @@
 %% @hidden
 -module(ra_log_snapshot).
 
+-behaviour(ra_snapshot).
+
 -export([
-         write/2,
+         prepare/2,
+         write/3,
+         save/3,
          read/1,
+         install/2,
```

Comment:
As discussed I'd like `install/2` removed in favour of just using `recover/1` - this will help when we want to implement chunked transfer that doesn't maintain the full state in memory.